app/cdap/components/GridTable/index.tsx

```diff
@@ -116,97 +116,42 @@ const GridTable = () => {
     }
   };
 
-  const convertNonNullPercent = (key, nonNullValue) => {
-    const lengthOfData = gridData.values.length;
-    let count = 0;
-    let nonNull: any = 0;
-    let empty: any = 0;
-    let nullValue: any = 0;
-    if (lengthOfData) {
-      nonNull = nonNullValue['non-null'] ? (nonNullValue['non-null'] / 100) * lengthOfData : 0;
-      nullValue = nonNullValue.null ? (nonNullValue.null / 100) * lengthOfData : 0;
-      empty = nonNullValue.empty ? (nonNullValue.empty / 100) * lengthOfData : 0;
-      count = parseInt(nullValue + empty);
-    }
-    return count;
-  };
-
-  const checkFrequentlyOccuredValues = (key) => {
-    const valueOfKey = gridData.values.map((el) => el[key]);
-    let mostfrequentItem = 1;
-    let count = 0;
-    let item = '';
-    const data = {
-      name: '',
-      count: 0,
-    };
-    for (let i = 0; i < valueOfKey.length; i++) {
-      for (let j = i; j < valueOfKey.length; j++) {
-        if (valueOfKey[i] == valueOfKey[j]) {
-          count++;
-        }
-        if (mostfrequentItem < count) {
-          mostfrequentItem = count;
-          item = valueOfKey[i];
-        }
-      }
-      count = 0;
-      item = item == '' ? valueOfKey[i] : item;
-    }
-    data.name = item;
-    data.count = mostfrequentItem;
-    return data;
-  };
-
-  const createMissingData = (statistics) => {
-    const objectArray = Object.entries(statistics);
+  const createMissingData = (statistics: IDataOfStatistics) => {
+    const statisticObjectToArray = Object.entries(statistics);
```

Comment:
Yes, because variables are changed to meaningful name

Reply:
That makes it hard to review, because it's not clear which PR has the "real" changes. If you have 1 PR that's dependent on another, can you raise them 1 at a time?

Also, it looks like there are no commits in common, despite the common changes. That might make things harder at merge time.